app/views/accounts.py

```diff
@@ -28,7 +28,7 @@ def login():
     form = LoginForm()
     if request.method == 'POST':
         try:
-            user = User.authenticat(form.email.data, form.password.data)
+            user = User.authenticat(User.get(email=form.email.data), form.email.data, form.password.data)
```

Comment:
you don't need to pass it a user instance it gets passed to the function as the first argument by default if it's an object method